assets/js/base/components/cart-checkout/order-summary/order-summary-item.js

```diff
@@ -151,6 +170,17 @@ const OrderSummaryItem = ( { cartItem } ) => {
 					variation={ variation }
 				/>
 			</div>
+			<span className="screen-reader-text" tabIndex={ 0 }>
+				{ sprintf(
+					/* translators: Total Price for %d quantity items: %s */
```

Comment:
In the translators comment, you need to explain what each variable will be replaced with. For example, in this case it could be written as:
```suggestion
					/* translators: %1$d is the number of items and %2$s is the total prices including the currency symbol. */
```

Reply:
Thanks for the suggestion, I've adjusted the items with the appropriate description.